runtime/compiler/codegen/J9WatchedInstanceFieldSnippet.cpp

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
```

Comment:
The copyright for both new files needs to be `2019, 2019`. Similarly in the other files.

Reply:
f0eae7d